src/6_branch.js

```diff
@@ -1013,7 +1013,7 @@ Branch.prototype['sendSMS'] = wrap(
 					self._api(
 						resources.linkClick,
 						{
-							"link_url": 'l/' + url.split('/').pop(),
```

Comment:
sorry, i am not familiar with the context here (first time review web sdk code), but what do we do with the link domain here?